src/test/resources/application-test-containers.yaml

```diff
@@ -0,0 +1,4 @@
+spring:
+  liquibase:
+    change-log: classpath:/liquibase/changelog.xml
+    show-summary: summary
```

Comment:
We don't need this profile since it's only and always needed when using the ExtendWithSwatchDatabase. Instead, we could add these properties directly in https://github.com/RedHatInsights/rhsm-subscriptions/blob/8d46fa82d826ffc9b76e6eab6864243a725449f4/src/test/java/org/candlepin/subscriptions/test/ExtendWithSwatchDatabase.java#L40.

Reply:
Removed